extensions/wikia/EditPageLayout/js/plugins/SyntaxHighlighter.js

```diff
@@ -0,0 +1,563 @@
+//Syntax highlighter with various advantages
+//See [[User:Remember the dot/Syntax highlighter]] for more information
+
+define('WikiTextSyntaxHighlighter', function() {
+	var init = function (textarea) {
```

Comment:
I guess this could be 
function init (textarea) {...}
to be consistent with the rest of the module methods